app/code/Dev/ProductComments/view/frontend/templates/product/view/comments.phtml

```diff
@@ -0,0 +1,70 @@
+    </div>
+    <?php foreach ($comments as $item): ?>
+    <div class="block-content">
+        <ol class="items review-items">
+            <li class="item review-item" itemscope="" itemprop="review" itemtype="http://schema.org/Review">
+                <div class="review-content" itemprop="description"><?= $item->getData("body") ?></div>
+                <div class="review-details">
+                    <p class="review-author"></p>
+                    <p><b><?= $item->getData('comment')?></b></p>
+                    <span class="review-details-label">Commented by</span>
+                    <span class="review-details-value"
+                          itemprop="author"><?= $item->getData("name") ?></span>
+                    <br>
+                    <p class="review-date">
+                        <span class="review-details-label">Posted on</span>
+                        <time class="review-details-value"
+                              itemprop="datePublished"><?= date('m/j/Y', strtotime($item->getData("date"))) ?></time>
+                    </p>
+                </div>
+            </li>
+        </ol>
+    </div>
+    <?php endforeach; ?>
+</div>
+<?php endif;
+if($commentsAvailable === 'yes') :
+    ?>
+    <form class="form" id="custom-form" action="<?=$block->getUrl('commentvisibility/index/sendComment')?>" data-mage-init='{"validation": {}}' method="post" autocomplete="off">
+        <fieldset class="fieldset">
+            <div class="field required">
+                <label for="name" class="label"><span><?= __('Name') ?></span></label>
+                <div class="control">
+                    <input type="text" name="name" id="name" value="" title="<?= __('Name') ?>" class="input-text" data-validate="{required:false}">
+                </div>
+            </div>
+            <div class="field required">
+                <label for="email_address" class="label"><span><?= __('Email') ?></span></label>
+                <div class="control">
+                    <input type="email" name="email" id="email_address" value="" title="<?= __('Email') ?>" class="input-text" data-validate="{required:true, 'validate-email':true}">
+                </div>
+            </div>
+            <div class="field required">
+                <label for="comment" class="label"><?= __('Comment') ?></label>
+                <textarea name="comment" id="comment" value="" title="<?= __('Comment') ?>" class="input-text" data-validate="{required:true}"></textarea>
+            </div>
+        </fieldset>
+        <input type="hidden" id="product_id" name="productId" value="<?= $productId ?>">
+        <div class="actions-toolbar">
+            <div class="primary">
+                <button type="submit" class="action submit primary" title="<?= __('Submit') ?>"><span><?= __('Submit') ?></span></button>
+            </div>
+        </div>
+    </form>
+<?php else : ?>
+    <p> Comments ara  Disabled for this Product </p>
```

Comment:
Make string translatable and fix typo. https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/translate_theory.html

Reply:
```<item name="label" xsi:type="string" translate="true">Comments are Disabled</item>```